pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -791,6 +791,11 @@ protected void validateGlobalNamespaceOwnership(NamespaceName namespace) {
 
                     log.warn(msg);
                     validationFuture.completeExceptionally(new RestException(Status.PRECONDITION_FAILED, msg));
+                } else if (policies.deleted) {
```

Comment:
Should we check this `deleted` mark first, then the rest conditions in L774 and L780?

Reply:
Hi, I think we should check this `deleted` mark at the last, because namespace deletion is a low-frequency operation, so it should be checked later to reduce the judgment.  What do you think?

Reply:
@Technoboy- 
My concern is more about if we delete a namespace, we may still get previous error info "Namespace does not have any clusters configured : local_cluster=%s ns=%s". 
It's confusing for users.

> so it should be checked later to reduce the judgment.

The performance issue here is insignificant.

Reply:
+1

Check the namespace delete or not in advance is more reasonable here, after users deleted a namespace, they should not get errors like "Namespace does not have any clusters configured".

Reply:
> @Technoboy- My concern is more about if we delete a namespace, we may still get previous error info "Namespace does not have any clusters configured : local_cluster=%s ns=%s". It's confusing for users.
> 
> > so it should be checked later to reduce the judgment.
> 
> The performance issue here is insignificant.

yes, agree.

Reply:
fixed, thanks @Jason918 @codelipenghui @shibd .